_layouts/docwithnav.html

```diff
@@ -202,10 +202,16 @@
               <a data-proofer-ignore href="https://github.com/projectcalico/calico/tree/master/{{page.path}}" style="color: #999; text-decoration: none; float: right;" target="_blank"><i class="fa fa-github" aria-hidden="true"></i> Edit this page</a></div>
               <!-- END TOP DOCUMENTATION PAGE NAVBAR -->
               {% include version_warning.html %}
+              {% if page.calico_enterprise -%}
+              {% include enterprise_header.html %}   
```

Comment:
Want to just put the contents of the 'includes' for the header and footer here instead of introducing more include files?

Reply:
I did that originally, but realized that they may become arbitrarily more complex, and it was starting to look messy.  So thought better to isolate them into separate include files. I'm more comfortable with people editing those separate files rather than them messing with this main template.